platform/Http.roc

```diff
@@ -85,3 +86,11 @@ get! = \uri, fmt ->
 
     Decode.fromBytes response.body fmt
     |> Result.mapErr \_ -> HttpDecodingFailed
+
+get_utf8! : Str => Result Str [BadBody Str]
```

Comment:
It doesn't seem like we have this error union anymore for errors, so we should maybe provide an error tag because it's lower cardinality? I'd vote `[InvalidUtf8Body]` instead.